client/components/WordInfoCard.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { Card, Icon, List, Header } from 'semantic-ui-react';
+
+/**
+ * COMPONENT STYLE
+ */
+const style = {
+};
+
+/**
+ * COMPONENT
+ */
+export const UserHome = props => {
+  const { word } = props;
```

Comment:
Jung, it looks like you're pulling 'word' from props but you didn't pull 'word' from the store in your mapState function down at line 71.  If you have tested this and it works the way you expect that's fine.